skrifa/src/string.rs

```diff
@@ -597,12 +597,72 @@ const LANGUAGE_ID_TO_BCP47: &[(u16, &str)] = &[
     (0x540a, "es-US"),        //Spanish
 ];
 
+fn _get_best_name(font: &FontRef, name_ids: Vec<StringId>) -> Option<String> {
+    for name_id in name_ids {
+        if let Some(name) = LocalizedStrings::new(font, name_id).english_or_first() {
+            return Some(name.to_string());
+        }
+    }
+    None
+}
+
+pub fn get_best_family_name(font: &FontRef) -> Option<String> {
+    let name_ids = vec![
+        StringId::WWS_FAMILY_NAME,
+        StringId::TYPOGRAPHIC_FAMILY_NAME,
+        StringId::FAMILY_NAME,
+    ];
```

Comment:
instead of `vec!` (which allocates) this should just be a const array:

```rust
const NAME_IDS: [StringId; 3] = [
    StringId::WWS_FAMILY_NAME,
    StringId::TYPOGRAPHIC_FAMILY_NAME,
    StringId::FAMILY_NAME,
];